site/en/tutorials/audio/transfer_learning_audio.ipynb

```diff
@@ -345,7 +342,7 @@
         "\n",
         "and all the audio files are in `./datasets/ESC-50-master/audio/`\n",
         "\n",
-        "You will create a pandas dataframe with the mapping and use that to have a clearer view of the data.\n"
+        "You will create a pandas DataFrame with the mapping and use that to have a clearer view of the data.\n"
```

Comment:
backticks for code. `DataFrame`

Reply:
That's a good point @MarkDaoust . Will add them here. 

Can we use both like pandas does? https://pandas.pydata.org/pandas-docs/stable/user_guide/dsintro.html#dataframe